sdk/sdk-api/src/main/java/mil/emp3/api/Overlay.java

```diff
@@ -42,41 +42,65 @@ public Overlay(IGeoContainer geoContainer) {
     }
 
     @Override
-    public void addOverlay(IOverlay overlay, boolean visible)
+    public void addOverlay(IOverlay overlay, boolean visible, Object object)
```

Comment:
Considering that this will NOT work for Camera events, we should limit the changes for Container add/remove capability.